test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 2950
+prepare_write_cmd: [
+  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
```

Comment:
@fruch what does workloada do?
Why not running more workloads in the test?

Reply:
workloada is doing 50% read / 50% write.

https://github.com/brianfrankcooper/YCSB/wiki/Core-Workloads